Lib/compileall.py

```diff
@@ -367,9 +367,9 @@ def main():
                               'environment variable is set, and '
                               '"timestamp" otherwise.'))
     parser.add_argument('-o', action='append', type=int, dest='opt_levels',
```

Comment:
Maybe fix the `dest=` here too to be singular.

Reply:
It is actually a list.
I've fixed the fixup.